RandomQuestion/MainWindow.xaml

```diff
@@ -12,11 +11,16 @@
             <RowDefinition Height="28"/>
             <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
-        <StackPanel x:Name="TitleBar"  Grid.Row="0" Margin="0, 0, 120, 0"
```

Comment:
Should split these on multiple rows. Same for all others in this file

Reply:
Oh git, crazy git! That line is actually removed (-) and was replaced by what we see with (+).